packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -84,7 +84,7 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
           data-is-focusable={ !disabled }
           ref={ (c): HTMLElement => this._dropDown = c }
           id={ id }
-          className={ css('ms-Dropdown', {
+          className={ css((this.props.className), 'ms-Dropdown', {
```

Comment:
can you just add className to the list of destructures from this.props above on line 69? Then you can just refer to className. Also the extra parenthesis are unnecessary.

also can you put it as the 2nd param, just for clarity? (It doesn't affect anything really, just that ms-Dropdown is the more important thing to indicate what the css is for.)

```ts
css('ms-Dropdown', className, {
```